src/webots/vrml/WbProtoManager.hpp

```diff
@@ -113,14 +117,14 @@ class WbProtoManager : public QObject {
   enum { BASE_NODE = 10001, PROTO_WORLD = 10002, PROTO_PROJECT = 10003, PROTO_EXTRA = 10004, PROTO_WEBOTS = 10005 };
 
   // given a category and a PROTO name, it returns the url from the corresponding list
-  QString protoUrl(int category, const QString &protoName) const;
+  QString protoUrl(const QString &protoName, int category);
```

Comment:
Why did you remove the const qualifier? It seems this method doesn't modify the object:
```suggestion
  QString protoUrl(const QString &protoName, int category) const;
```